src/azure-cli/azure/cli/command_modules/resource/custom.py

```diff
@@ -2268,6 +2270,8 @@ def resolve_api_version(rcf, resource_provider_namespace, parent_resource_path,
     @staticmethod
     def _resolve_api_version_by_id(rcf, resource_id):
         parts = parse_resource_id(resource_id)
+        if 'namespace' not in parts:
```

Comment:
use more detailed condition for resource group

Reply:
Done